docs/book/src/usage.md

```diff
@@ -1,3 +1,29 @@
 # Usage
 
 This section is intended for users who want to install Marker and run provided lints.
+
+## Controlling Marker Lints
+Once your crate is configured to run some lints, it quickly becomes important to control the lint levels within your
+code. Marker provides the ability to use normal lint control attributes like `#[allow(...)]` `#[deny(...)]` and others
+to control the behavior of marker lints.
+
+### On nightly
+Marker provides this functionality via the `#[register_tool()]` attribute currently available in
+[nightly](https://github.com/rust-lang/rust/issues/66079). If your crate is compiled using nightly, then controlling
+lints is as simple as placing `#![feature(register_tool)]` and `#![register_tool(marker)]` at the top of your crate
+`lib.rs` or `mod.rs` file. You can then use all of the normal lint attributes you might usually use, on marker provided
+lints, like `#[allow(marker::my_lint)]`.
```

Comment:
I would make this statement a bit stronger, saying that rustc requires the feature. Maybe:

```md
Marker's lints use the `marker::` tool prefix. Rustc requires the [`register_tool` feature](https://github.com/rust-lang/rust/issues/66079) to allow this prefix in the normal lint level attributes, like `#[allow(...)]` and `#[deny(...)]`....
```

> Marker's lints use the `marker::` tool prefix. Rustc requires the [`register_tool` feature](https://github.com/rust-lang/rust/issues/66079) to allow this prefix in the normal lint level attributes, like `#[allow(...)]` and `#[deny(...)]`....

It's likely that most projects use a stable version by default. It would be nice to have a small code block, which could be directly copied. This should probably work:

```rs
#[cfg_attr(marker, feature(register_tool))]
#[cfg_attr(marker, register_tool(marker))]
```